finviz/screener.py

```diff
@@ -13,6 +13,57 @@
 class Screener(object):
     """ Used to download data from http://www.finviz.com/screener.ashx. """
 
+    @classmethod
+    def init_from_url(cls, url, rows=None):
+        """
+        Initilizes from url
+
+        :param url: screener url
+        :type url: string
+        :param rows: total number of rows to get
+        :type rows: int
+        """
+
+        splitted_url = urlparse(url)
+        splitted_query = urlparse_qs(splitted_url.query)
+
+        tickers = None
+        if 't' in splitted_query:
+            tickers = splitted_query['t'][0].split(',')
+
+        filters = None
+        if 'f' in splitted_query:
+            filters = splitted_query['f'][0].split(',')
+
+        table = None
+        if 'v' in splitted_query:
+            table_numbers_types = {
+                '110': 'Overview',
+                '120': 'Valuation',
+                '130': 'Ownership',
+                '140': 'Performance',
+                '150': 'Custom',
+                '160': 'Financial',
+                '170': 'Technical'
+            }
+            table_number_string = splitted_query['v'][0][0:2] + '0'
+            try:
+                table = table_numbers_types[table_number_string]
+            except KeyError:
+                raise InvalidTableType(splitted_query['v'][0])
+        else:
+            table = 'Overview'
+
+        order = ''
+        if 'o' in splitted_query:
+            order = splitted_query['o'][0]
+
+        signal = ''
+        if 's' in splitted_query:
+            order = splitted_query['s'][0]
```

Comment:
Just curious, why is `order = splitted_query['s'][0]` here? Should it be `signal = ...`? Otherwise, `signal` will always return `''`?

Reply:
Indeed, it is a typo. I just made pull request to fix the bug here: #33 
Thanks for the notice